.github/workflows/test-deploy-website.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         with:
           dotnet-version: '6.0.x'
 
-      - run: dotnet tool install --global gsferreira.XmlDocMarkdown.Docusaurus --version 0.0.1-beta1 # using this version while the Pull Request isn't accepted here: https://github.com/ejball/XmlDocMarkdown/pull/126
+      - run: dotnet tool install --global gsferreira.XmlDocMarkdown.Docusaurus --version 0.0.1-beta2 # using this version while the Pull Request isn't accepted here: https://github.com/ejball/XmlDocMarkdown/pull/126
```

Comment:
Is expected to have the `gsferreira` prefix in the --global parameter?

Reply:
Yes. It's a temporary package until the original library accepts the PR since the dotnet tool has some format issues.